java/src/main/java/org/rocksdb/BackupableDBOptions.java

```diff
@@ -264,9 +448,18 @@ public boolean shareFilesWithChecksum() {
   private native boolean backupLogFiles(long handle);
   private native void setBackupRateLimit(long handle, long rateLimit);
   private native long backupRateLimit(long handle);
+  private native void setBackupRateLimiter(long handle, long rateLimiterHandle);
   private native void setRestoreRateLimit(long handle, long rateLimit);
   private native long restoreRateLimit(long handle);
+  private native void setRestoreRateLimiter(final long handle,
+      final long rateLimiterHandle);
   private native void setShareFilesWithChecksum(long handle, boolean flag);
   private native boolean shareFilesWithChecksum(long handle);
+  private native void setMaxBackgroundOperations(final long handle,
+      final int maxBackgroundOperations);
+  private native int maxBackgroundOperations(final long handle);
+  private native void setCallbackTriggerIntervalSize(final long handle,
```

Comment:
Curious ... Why are some params final whereas others are non-final (in this whole file, and in a few other places)? Is it just that all the newer additions will be final going forward?